src/govuk/components/character-count/character-count.yaml

```
@@ -76,6 +72,34 @@ params:
     type: string
     required: false
     description: Classes to add to the count message.
+- name: textareaDescriptionText
+  type: string
+  required: false
+  description: Message made available to assistive technologies to describe that the component accepts only a limited amount of content. It is visible on the page when JavaScript is unavailable. The component will replace the `%{count}` placeholder with the value of the `maxlength` or `maxwords` parameter.
+- name: charactersUnderLimitText
+  type: string
```

Possibly misleading for pluralised lists – should this be object? (Same for all pluralised messages)

```suggestion
  type: object
```

---

Indeed, good catch! Got too focused on the content of the messages and forgot to check the types 😬